lib/sorcery/providers/twitter.rb

```diff
@@ -15,7 +15,7 @@ def initialize
         super
 
         @site           = 'https://api.twitter.com'
-        @user_info_path = '/1.1/account/verify_credentials.json'
+        @user_info_path = '/1.1/account/verify_credentials.json?include_email=true'
```

Comment:
Including the email is good, but iirc it requires additional permissions on the app config side. I'm worried that this could potentially break existing applications, and this should probably be a optional config in-of-itself. Unfortunately though, I don't have the time currently to investigate myself.

@ebihara99999 or @EvanBrightside, can you look into how this interacts with existing apps that don't have the email permission checked? If we can confirm that it doesn't break existing apps, and/or update this to be configurable from `initializer.rb`, then I'd feel more comfortable merging. Thanks!

Reply:
Thank you for the review. As I said I think the parameter should be optional too because it returns nil, email is nil, if the app doesn’t have granted by Twitter. It breaks not-null constraint, which is imposed almost all cases.

Reply:
@EvanBrightside

Would you change codes into configurable in initializer?

See this PR 
https://github.com/Sorcery/sorcery/pull/109/files

And the initializer is lib/generators/sorcery/templates/initializer.rb

Reply:
If it’s somehow difficult or you don’t understand very well, mention to me. Just typing with a mobile, the explanation may be too short.